integration-tests/ws-security-policy/src/main/resources/application.properties

```diff
@@ -70,6 +70,8 @@ quarkus.cxf.endpoint."/helloSaml1".security.signature.properties."org.apache.ws.
 quarkus.cxf.endpoint."/helloSaml1".security.signature.properties."org.apache.ws.security.crypto.merlin.keystore.password" = password
 quarkus.cxf.endpoint."/helloSaml1".security.signature.properties."org.apache.ws.security.crypto.merlin.keystore.alias" = bob
 quarkus.cxf.endpoint."/helloSaml1".security.signature.properties."org.apache.ws.security.crypto.merlin.file" = bob.${keystore.type}
+quarkus.cxf.endpoint."/helloSaml1".security.saml-callback-handler = #saml1CallbackHandler
+
 
```

Comment:
Both client and server need the samlCallbackHandler to insert saml token(which needs to be signed) because in the test we share the same policy xml between the client and server.